### DISCUSSION TITLE
Full renovate docker image not being published

### DISCUSSION THREAD

**Discussion by rzuckerm** [renovatebot/renovate#17526] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.169.0

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

3.5.3

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

The renovate docker images are not being published for the full image to docker hub. It looks like the latest image that was posted is [32.185.0-slim](https://hub.docker.com/layers/renovate/renovate/renovate/32.185.0-slim/images/sha256-fd89dc6092ab1fc5551e0e4b93a72ae7b447c58b498e9b08f68209c828db246d?context=explore), but there is no corresponding full image. It looks like the last full image that was posted is [32.182.4](https://hub.docker.com/layers/renovate/renovate/renovate/32.182.4/images/sha256-5e3fc0e2a0d704961a3c50a3edc0f2a5b928185b22fa5d75686062f6753d526d?context=explore),

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment, marked as answer]
Wrong, please don't create any more bug reports in future

**Reply by rzuckerm** [reply]
Why not?

**Reply by viceice** [reply]
It's not a bug, the full image needs some time longer to publish than the slim image
![image](https://user-images.githubusercontent.com/1798109/187700920-4664158d-7031-458b-a7f6-9c284fcecd32.png)
